src/discrete-layer/components/map-container/highlighted-layer.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { CesiumGeojsonLayer, CesiumColor } from '@map-colonies/react-components';
+import { FeatureCollection } from 'geojson';
+import { ConstantProperty } from 'cesium';
+import { useStore } from '../../models/rootStore';
+import { getLayerFootprint } from '../../models/layerImage';
+
+const FOOTPRINT_BORDER_COLOR = CesiumColor.BLUE;
+const FOOTPRINT_BORDER_WIDTH = 12.0;
+
+export const HighlightedLayer: React.FC = observer(() => {
+  const { discreteLayersStore } = useStore();
+  const [layersFootprints, setlayersFootprints] = useState<FeatureCollection>();
+  useEffect(() => {
+    const footprintsCollection: FeatureCollection = {
+      type: 'FeatureCollection',
+      features: []
+    }
+    const layer = discreteLayersStore.highlightedLayer;
```

Comment:
discreteLayersStore.highlightedLayer is a method i'd prefer you call it as one: getFoo() instead of a property signature

Reply:
It's a property, not a method
```
export const discreteLayersStore = types
  .model({
    state: types.enumeration<ResponseState>(
      'State',
      Object.values(ResponseState)
    ),
    searchParams: types.optional(searchParams, {}),
    layersImages: types.maybe(types.frozen<LayersImagesResponse>([])),
    highlightedLayer: types.maybe(types.frozen<ILayerImage>()),             /// <---- PROPERTY
  })
```